docs/source/workflow_inputs.rst

```diff
@@ -2144,6 +2181,63 @@ If a backup type of "separate" is provided, additional information is entered in
     - There cannot be multiple backup heating systems.
 
+.. _geothermal_loops:
+
+HPXML Geothermal Loops
+**********************
+
+Each geothermal loop is entered as an ``/HPXML/Building/BuildingDetails/Systems/HVAC/HVACPlant/GeothermalLoop``.
+
+  ========================================  ================  ===========  ===============  ========  ==============  ===============================================
+  Element                                   Type              Units        Constraints      Required  Default         Notes
+  ========================================  ================  ===========  ===============  ========  ==============  ===============================================
+  ``SystemIdentifier``                      id                                              Yes                       Unique identifier
+  ``LoopConfiguration``                     string                         vertical         Yes
+  ``LoopFlow``                              double            gal/min      > 0              No        autosized [#]_  Water flow rate through the geothermal loop
+  ``BoreholesOrTrenches/Count``             integer                        > 0 [#]_         No        autosized [#]_
+  ``BoreholesOrTrenches/Length``            double            ft           See [#]_         No        autosized [#]_  Length (i.e., average depth) of each borehole
+  ``BoreholesOrTrenches/Spacing``           double            ft           > 0              No        16.4            Distance between boreholes
+  ``BoreholesOrTrenches/Diameter``          double            in           > 0              No        5.0
+  ``Grout/Type`` or ``Grout/Conductivity``  string or double  Btu/hr-ft-F  See [#]_ or > 0  No        standard        Grout type or conductivity [#]_
+  ``Pipe/Type`` or ``Pipe/Conductivity``    string or double  Btu/hr-ft-F  See [#]_ or > 0  No        standard        Pipe type or conductivity [#]_
+  ``Pipe/Diameter``                         double            in           See [#]_         No        0.75
+  ``Pipe/ShankSpacing``                     double            in           > 0              No        See [#]_        Center-to-center distance between two branches of a vertical U-tube
+  ``extension/BorefieldConfiguration``      string                         See [#]_         No        Rectangle
+  ========================================  ================  ===========  ===============  ========  ==============  ===============================================
+
+  .. [#] LoopFlow autosized per TODO.
+  .. [#] | If extension/BorefieldConfiguration provided, a valid BoreholesOrTrenches/Count must also be provided:
+         | - **Rectangle**: 1, 2, 3, 4, 5, 6, 7, 8, 9, or 10
+         | - **Open Rectangle**: 8 or 10
+         | - **C**: 7 or 9
+         | - **L**: 4, 5, 6, 7, 8, 9, or 10
+         | - **U**: 7, 9, or 10
+         | - **Lopsided U**: 6, 7, 8, 9, or 10
```

Comment:
Any use of "|" in the docs will need to be changed per https://github.com/NREL/OpenStudio-HPXML/pull/1544